deploy/rbac-clowdapp.yml

```diff
@@ -8,6 +8,8 @@ objects:
   kind: ClowdApp
   metadata:
     name: rbac
+    annotations:
+      bonfire.dependencies: "model-access-permissions-yml-stage,rbac-config-yml-stage"
```

Comment:
Can we make this a template parameter?

So:
```
bonfire.dependencies: ${BONFIRE_DEPENDENCIES}
```

And down at the bottom under `parameters` section:
```
- name: BONFIRE_DEPENDENCIES
  value: "model-access-permissions-yml-stage,rbac-config-yml-stage"
```

Reply:
:+1: